doc/tutorials/rtvm_performance_tips.rst

```diff
@@ -0,0 +1,213 @@
+.. _rt_perf_tips_rtvm:
+
+ACRN Real-Time VM Performance Tips
+##################################
+
+Background
+**********
+
+The ACRN real-time VM (RTVM) is a special type of ACRN post-launched VM. In
+order to achieve bare metal-like RT performance, a set of constraints and
+technologies are applied to the RTVM compared to the ACRN standard VM. With
+these additional constraints and technologies, RT tasks can run on the RTVM
+without a VM-exit, which is a key virtualization overhead issue.

```

Comment:
I think a better phrasing would be: "RT tasks can run in an RTVM without incurring `VM-exit` instructions. Those are a primary source of performance overhead when running in a virtualized environment". (or something similar)